Makefile

```diff
@@ -74,3 +74,8 @@ lint-rust:
 
 prettier:
 	npx prettier --write .
+
+### Deploy ###
+
+deploy:
+	forge script ./script/deploy/Deploy.s.sol --broadcast --slow --verify --resume --multi --isolate --legacy
```

Comment:
Is this going to be added, or should we delete this?

Reply:
it's gonna go